app/views/applications/new.html.erb

```diff
@@ -40,6 +55,30 @@
 
   <div class="form_field">
     <p>
+      <%= f.check_box :terms_and_conditions %>
+      <%= f.label :terms_and_conditions, "Terms and Conditions" %>
```

Comment:
just a minor improvement: could you change this to either
"Terms and Conditions:"
or
"I agree with the following Terms and Conditions:"

basically to make the checkbox more understandable :)